projects/ui-components/lf-repository-browser/lf-repository-browser.component.ts

```diff
@@ -459,16 +465,17 @@ export class LfRepositoryBrowserComponent implements OnDestroy {
     if (parentEntry && parentEntry.id) {
       try {
         this.isLoading = true;
+        this.hasError = false;
```

Comment:
this function is called as part of initAsync, but also openChildFolderAsync, so I add a separate try/catch for it